src/index.js

```diff
@@ -78,6 +79,14 @@ export default class VerificationInput extends PureComponent {
     this.setSelection(this.state.selectedIndex + offset);
   }
 
+  saveInputRef = ref => {
+    this.input = ref
+    
+    if (this.props.getInputRef) {
```

Comment:
I think this should be initialized as `() => {}` in `defaultProps` and this check can be removed. What do you think?